content/management/configuration-api/v3.4/index.mdx

```diff
@@ -4337,7 +4338,7 @@ Informs that an agent's account was deleted.
 
 ### `auto_access_added`
 
-Informs that a new auto access rule was added.
+Informs that a new auto access rule was added. Response will contain full state of auto access rule resource - empty fields won't be omitted. Adding rule may result in multiple [`auto_access_updated`](#auto_access_updated) webhooks, depending on `next_id` value.
```

Comment:
`auto access rule resource` -> `the auto access rule resource`
`empty fields won't be omitted` -> this is somewhat cryptic, I'd put it simpler: `including not defined fields` or sth like that. Overall, I'm not sure this needs to be specified in the docs - as a dev, I would just expect all the fields, initialised or otherwise.
`Adding rule` -> `Adding a rule`
`depending on next_id value` -> `depending on the value of next_id` or simply `depending on next_id`

Reply:
1. We tend to use "data structure" instead of "resource".
2. "Empty fields won't be omitted" -> "including empty fields", "including not defined fields".
3. If we're going to add the additional info, I'd link the explanation of `next_id` as I forgot what it was and didn't see the direct connection between the number of webhooks and the value of `next_id`. I also wonder if we should include it as it seems super additional. What happens if we don't? 🤓